### PR TITLE
docs(examples): adds new example to repo

### PR DESCRIPTION
**Pull request by emnul**
# Summary

Creates a new example in the repository that demonstrates how to use the Ethereum Attestation Service with a Defender Relay Signer

### CODE REVIEW

File: examples/eas-relayer-signer/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import { EAS, NO_EXPIRATION, SchemaEncoder, TransactionSigner } from '@ethereum-attestation-service/eas-sdk';
+
+import dotenv from 'dotenv';
+dotenv.config();
+import { Agent } from 'node:https';
+import { env } from 'node:process';
```
Comment (MCarlomagno) [inline examples/eas-relayer-signer/src/index.ts:6]: We can remove this one and use `process.env` instead

**Review by MCarlomagno — APPROVED** [review]

**Comment by RaphaelMagnux** [comment]
Hello!

I see that for onchain attestation there is not type 🥲

Any idea about what would be?

Screenshot 2024-11-27 at 1 45 00 p m

**Comment by emnul** [comment]
@RaphaelMagnux Hi the type seems to be `AttestationRequest` found [here](https://github.com/ethereum-attestation-service/eas-sdk/blob/a704273f6dd11e460e7d98be7b939a546ed8f3ec/src/request.ts#L14)

**Comment by RaphaelMagnux** [comment]
Thank you, I already found a solution, instead use **EAS SDK**, I´m calling directly to EAS.sol **smart contract** with ethers and OZ Relayer, it works.

Thanksssss

**Review by CoveMB — APPROVED** [review]

**Pull request merged**